src/constants.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
     vote: 'Vote',
     wallet: 'Wallet',
     myActivities: 'MyActivities',
-    resultSetting: 'ResultSetting',
+    resultSet: 'ResultSetting',
```

Comment:
I think eventually we should change this to reflect it's value like the rest of these

Reply:
I have an issue open to refactor all these. we should just use ALL_CAPS for key/value.